scenarios/turbine-heat-sensor/gradle.properties

```diff
@@ -0,0 +1,14 @@
+/*
```

Comment:
Same comment about the license header format for this file.

Reply:
Added.